libs/database/src/lib/entities/site-name.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { AutoIncrement, Column, Model, PrimaryKey, Table } from "sequelize-typescript";
+import { BIGINT } from "sequelize";
+
+// A quick stub for the research endpoints
+@Table({ tableName: "v2_sites", underscored: true, paranoid: true })
```

Comment:
We already have a `site.entity.ts` that reflects this table. I can't think of any reason why we'd want to have two, even if sequelize is able to handle that correctly (I suspect we'd run into problems when v3 eventually becomes the source of truth for the table schemas and is responsible for migrations)

Reply:
sorry, that file shouldnt reach here, it was one of my tests to try an entity with only name for the query. 
Deleted